src/Platform/Microsoft.Testing.Platform/Extensions/AbortForMaxFailedTestsExtension.cs

```diff
@@ -53,7 +58,21 @@ public AbortForMaxFailedTestsExtension(
     public string Description { get; } = PlatformResources.AbortForMaxFailedTestsDescription;
 
     /// <inheritdoc />
-    public Task<bool> IsEnabledAsync() => Task.FromResult(_maxFailedTests.HasValue && _capability is not null);
+    public async Task<bool> IsEnabledAsync()
+    {
+        if (!_maxFailedTests.HasValue)
+        {
+            return false;
+        }
+
+        if (_capability is null)
+        {
+            await _outputDevice.DisplayAsync(this, new ErrorMessageOutputDeviceData(PlatformResources.AbortForMaxFailedTestsCapabilityNotAvailable));
+            return false;
+        }
```

Comment:
Maybe we should hard fail instead (wdyt @MarcoRossignoli?). Pros to hardfail, forces you to fix. Cons for it, if you did it wrong and shipped, people cannot use your version (but maybe that's not "too bad").

Reply:
Crash